doc/classes/Control.xml

```diff
@@ -973,8 +973,12 @@
 		<member name="custom_minimum_size" type="Vector2" setter="set_custom_minimum_size" getter="get_custom_minimum_size" default="Vector2(0, 0)">
 			The minimum size of the node's bounding rectangle. If you set it to a value greater than (0, 0), the node's bounding rectangle will always have at least this size, even if its content is smaller. If it's set to (0, 0), the node sizes automatically to fit its content, be it a texture or child nodes.
 		</member>
+		<member name="focus_layer" type="int" setter="set_focus_layer" getter="get_focus_layer" default="0">
+			The focus layer for this control. Each layer will only navigate to other [Control]s in the same layer. Each layer can only have one focus at a time.
+			Use [code]gui_set_active_focus_layer[/code] to set the active focus layer that will be navigated.
+		</member>
 		<member name="focus_mode" type="int" setter="set_focus_mode" getter="get_focus_mode" enum="Control.FocusMode" default="0">
-			The focus access mode for the control (None, Click or All). Only one Control can be focused at the same time, and it will receive keyboard signals.
+			The focus access mode for the control (None, Click or All). Only one [Control] per layer can be focused at the same time, and it will receive keyboard signals.
```

Comment:
```suggestion
			The focus access mode for the control (None, Click or All). Only one [Control] per focus layer can be focused at the same time, and it will receive keyboard signals.
```